docs/foundamentals/content.md

```diff
@@ -77,7 +77,7 @@ The content manager uses the [JSON Schema](https://mozilla-services.github.io/re
 [Link to `react-jsonschema-form`](https://github.com/mozilla-services/react-jsonschema-form)
 
 
-#### Field `ummBloc` (_Optional_) <a class="toc" id="toc-field-uischema-https-github-com-mozilla-services-react-jsonschema-form-optional" href="#toc-field-uischema-https-github-com-mozilla-services-react-jsonschema-form-optional"></a>
+#### Field `renderer` (_Optional_) <a class="toc" id="toc-field-uischema-https-github-com-mozilla-services-react-jsonschema-form-optional" href="#toc-field-uischema-https-github-com-mozilla-services-react-jsonschema-form-optional"></a>
```

Comment:
The anchor here looks wrong, it's probably a systematic issue? Are they autogenerated?

Reply:
Wow you're right, I wonder where they are coming from :-/

Will change them in the new doc